app/Http/Controllers/WebhookController.php

```diff
@@ -18,4 +20,16 @@ public function handlePaymentSucceeded($payload): void
 
         dispatch(new ProcessPaymentSucceededJob($payload));
     }
+
+    protected function findOrCreateCustomer(string $passthrough)
+    {
+        $passthrough = json_decode($passthrough, true);
+
+        // The passthrough data comes from the shop front-end. We cannot trust it.
+        if (! is_array($passthrough) || $passthrough['billable_type'] !== User::class) {
```

Comment:
Maybe use `getMorhpClass` here in case we one day want to switch to a morph map? I can imagine we'd forget to adjust this